src/pf-notification-drawer/pf-notification-drawer.scss

```diff
@@ -0,0 +1,46 @@
+.drawer-pf {
```

Comment:
+1 - if the CSS is *not* specific to the Custom Element or the approach here, it should be sourced from PF. Maybe we can contribute this particular CSS upstream?

@dlabrecq why SASS vs LESS? I don't really know... ;)